packages/cli/src/commands/init.ts

```diff
@@ -68,14 +68,6 @@ export const initCommand = async (options: InitCommandOptions) => {
 
   const isTypescriptProject = await detectTypescriptProject(resolvedPath);
 
-  if (!isTypescriptProject) {
```

Comment:
This normally throws an error if the project is not using typescript.
We don't need to do this check anymore.